resources/tentative/complex-static-html/src/big-dom-generator/components/tree-area.jsx

```diff
@@ -0,0 +1,69 @@
+import { LCG } from 'random-seedable';
+import { useRef } from 'react';
+import { SEED, MAX_DEPTH, MAX_BREADTH, CHILD_PROB, TARGET_SIZE } from './../params';
+import ChevronRight from './../assets/Smock_ChevronRight_18_N.svg';
+
+const FolderWrapper = (props) => {
+  const {nodeCount, rando, maxDepth, maxBreadth, childProb, currentDepth} = props;
+  if (currentDepth >= maxDepth) {
+    return (null);
+  }
+  // Choose a random number of children.
+  const numChildren = rando.randRange(1, maxBreadth);
+  const children = [];
+  for (let i = 0; i < numChildren && nodeCount.current < TARGET_SIZE; i++) {
+    children.push(<TreeItem key={i}
+                            nodeCount={nodeCount}
```

Comment:
Nit: wrong indentation style.
It should be indented by exactly 4 spaces.

Reply:
@lpardosixtosMs - you can opt into our format options that we use from root, by adding exceptions into these files:
https://github.com/WebKit/Speedometer/blob/main/.eslintignore
https://github.com/WebKit/Speedometer/blob/main/.prettierignore

and running `npm run format`
https://github.com/WebKit/Speedometer/blob/main/package.json#L23

This should get you pretty close to the webkit code style.

Reply:
Added the exception and ran formatter, thanks!